docs/labels.md

```diff
@@ -7,7 +7,11 @@ There are two main types of labels that kops can create:
 
 Both are specified at the InstanceGroup level.
 
-A nice use for CloudLabels is to specify [AWS cost allocation tags](http://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/cost-alloc-tags.html)
+A nice use for CloudLabels is to specify [AWS cost allocation tags](http://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/cost-alloc-tags.html).
+
+A good use for nodeLables is to implement [nodeSelector labels](https://kubernetes
+.io/docs/concepts/configuration/assign-pod-node/#step-two-add-a-nodeselector-field-to-your-pod-configuration) that 
+survive [AWS EC2 auto scaling groups](https://aws.amazon.com/autoscaling/) replacing unhealthy or unreachable instances.
```

Comment:
The suggested change sounds good to me.  The intention was to contrast with manually adding the labels to the nodes, launch configurations and auto scaling groups and indicate nodeLables and cloudLabels are the right way to create labels.  Implementing nodeSelector labels is something I personally experienced difficulty implementing as I didn't realize the labels on the auto scaling groups didn't transfer to the node instances the ASG creates and didn't realize the labels I defined manually on the nodes are dropped by the ASG until the first time I performed a rolling update.